DBMigrator/VersionDiff.cs

```diff
@@ -30,23 +31,42 @@ public string UpgradeDiffText(List<DBVersion> diff)
             return str;
         }
 
-        public string DowngradeDiffText(List<DBVersion> diff)
+        public void LogDownGradeDiffText(List<DBVersion> diff, ILogger logger)
         {
-            var str = "";
+            var newLine = "\n      ";
+
+            diff.Reverse();
```

Comment:
Are we getting the desired result? We are reversing this collection first here and afterwards in Rollback(List<DBVersion> versionsToRollback)
It is a bit hard to follow when a method reverses the inputted list or even that the methods are expecting the lists to be ordered in the first place